src/App.js

```diff
@@ -22,33 +22,37 @@ const App = () => {
   const [hand, setHand] = useState([])
 
   const drawCard = () => {
-    // 1)
+    // 1) The draw variable generates a random card
     const draw = `${rank[Math.floor(Math.random() * rank.length)]} ${
       suit[Math.floor(Math.random() * suit.length)]
     }`
-    // 2)
+    // 2)This is a conditon that checks the draw variable. I vaguley understand that the === -1 is checking to if a specific card is in the array. And, if this condition is met.
     if (hand.indexOf(draw) === -1) {
-      // 3)
+      // 3)This will change the hand and add a new car to it.
       setHand([...hand, draw])
-      // 4)
+      // 4) This condition checks to if there is not 52 cards. And, if this condition is met.
     } else if (hand.length !== 52) {
-      // 5)
+      // 5)The draw card function is called and this will randomize the cards again so there won't be a repeat of a card.
       drawCard()
-      // 6)
+      // 6)This else is basically a catch all if the above conditions are not met.
     } else {
-      // 7)
+      // 7) This just alerts that all cards have been delt.
       alert("All cards have been dealt.")
     }
   }
   // No need to change anything ABOVE this line ^
 
-  const shuffle = () => {}
+  const shuffle = () => {
+    window.location.reload();
+  }
```

Comment:
While using window.location.reload() does refresh the browser and thus resets state, we don't have to (and shouldn't) use this functionality in React.  remember, the idea of React is to update only what's changing without refreshing the page.  Think about what needs to happen in the shuffle function and see if you can replace line 46... you can do it!